VagrantBuild.sh

```diff
@@ -37,18 +38,11 @@ if [ ! -f LocalConfig.pri ] && ! grep --quiet QMAKE_CXX LocalConfig.pri; then
 fi
 
 echo "Building Hoot... "
-echo "Will take several extra minutes to build the training data the initial time Hootenanny is installed only."
 make -s clean && make -sj$(nproc)
 
-# Waiting until Tomcat8 is installed
 # vagrant will auto start the tomcat service for us, so just copy the web app files w/o manipulating the server
-TOMCAT_USER=tomcat8
-VMUSER=`id -u -n`
-if [ "$VMUSER" = 'vagrant' ]
-then
-  TOMCAT_USER=$VMUSER
-fi
-sudo -u $TOMCAT_USER  scripts/tomcat/CopyWebAppsToTomcat.sh #&> /dev/null
+# Copy the web apps, no need to use sudo
+./scripts/tomcat/CopyWebAppsToTomcat.sh
```

Comment:
Moved all of the `TOMCAT_USER` stuff inside of the `CopyWebAppsToTomcat.sh` script so that it is self contained and doesn't need to be run with `sudo`.  Each call that needs it in the script has changed.  Should be easier to run that way.